### PR TITLE
pointing to TEST_URL

### PR DESCRIPTION
**Pull request by smathangi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by danlysiak — APPROVED** [review]

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -120,7 +120,7 @@ withPipeline("java", product, component) {
       }
       env.DEFINITION_STORE_HOST = "http://ccd-definition-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
       env.CCD_DATA_STORE_API_BASE_URL = "http://ccd-data-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
-      env.MCA_API_BASE_URL = "http://aac-manage-case-assignment-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
+      env.MCA_API_BASE_URL = env.TEST_URL
```
Comment (mattnayler) [inline Jenkinsfile_CNP:123]: NB: When this PR runs in the pipeline this edited line will not take effect unless the number of this PR is added to: `prsToUseAat` variable on line 15.  

Would it be possible/worth assigning `env.MCA_API_BASE_URL = env.TEST_URL` in all scenarios by moving this line to 71?

Comment (smathangi) [reply Jenkinsfile_CNP]: aha.. i didn't see it has multiple references. Fixed now @mattnayler

**Review by mattnayler — APPROVED** [review]
LGTM 👍

**Review by MSancaktutar — APPROVED** [review]
Lgtm

**Pull request merged**